src/Form/UserAccount/UserProfileType.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Form\UserAccount;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\EmailType;
+use Symfony\Component\Form\Extension\Core\Type\PasswordType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Component\Validator\Constraints\NotNull;
+
+class UserProfileType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('name', TextType::class, [
+                'required' => false,
+            ])
+            ->add('email', EmailType::class, [
+                'constraints' => [
+                    new NotNull(message: 'You must have an email address.'),
+                    new NotBlank(message: 'You must have an email address.'),
+                ],
+            ])
+            ->add('password', PasswordType::class, [
+                'required' => false,
+            ])
```

Comment:
Hmm, not sur ! 

1. you don't ask for the previous password
2. you don't encode the new password in the controller
3. if the password is empty, it erase the previous password

=> Usually, we create another form / endpoint / template to deal with password